python/fastfec.py

```diff
@@ -27,11 +27,16 @@ def __init__(self):
         # Initialize
         self.persistent_memory_context = self.libfastfec.newPersistentMemoryContext()
 
-    def parse(self, file_handle):
+    def parse(self, file_handle, include_filing_id=None, should_parse_date=True):
```

Comment:
So if it's `None` it's not included, and if it's a string that string gets inserted into the first column?

Reply:
Yea, that's exactly right